nixos/modules/config/swap.nix

```
@@ -114,6 +114,19 @@ let
         '';
       };
 
+      discardPolicy = mkOption {
+        default = null;
+        example = "once";
+        type = types.nullOr (types.enum ["once" "pages" "both" ]);
+        description = ''
+        Specify the discard policy for the swap device. Can be one of "once", "pages" or "both".
```

```suggestion
        Specify the discard policy for the swap device.
```
This is specified by the type. For instance in the man page:
`Type: null or one of "once", "pages", "both"`

---

Makes sense, thank you